enterprise/helpers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for enterprise app.
+"""
+
+from .models import EnterpriseCustomerBrandingConfiguration
+
+
+def get_branding_info(provider_id=None):
```

Comment:
Since this util method is for enterprise only so can we name it to something like `get_enterprise_branding_config`